src/test/java/teammates/test/cases/browsertests/InstructorHomePageUiTest.java

```diff
@@ -273,12 +273,47 @@ private void testRemindActions() {
         homePage.verifyStatus(Const.StatusMessages.FEEDBACK_SESSION_REMINDERSSENT);
         homePage.goToPreviousPage(InstructorHomePage.class);
 
-        ______TS("remind action: CLOSED feedback session");
 
-        homePage.verifyUnclickable(homePage.getRemindLink(feedbackSessionClosed.getCourseId(),
-                                                          feedbackSessionClosed.getFeedbackSessionName()));
-        homePage.verifyUnclickable(homePage.getRemindOptionsLink(feedbackSessionClosed.getCourseId(),
-                                                                 feedbackSessionClosed.getFeedbackSessionName()));
+
+        ______TS("remind action: CLOSED feedback session - inner button");
+
+        homePage.clickRemindOptionsLink(feedbackSessionClosed.getCourseId(), feedbackSessionClosed.getFeedbackSessionName());
+        homePage.clickAndCancel(homePage.getRemindInnerLink(feedbackSessionClosed.getCourseId(),
+                                                            feedbackSessionOpen.getFeedbackSessionName()));
+        homePage.clickRemindOptionsLink(feedbackSessionClosed.getCourseId(), feedbackSessionClosed.getFeedbackSessionName());
+        homePage.clickAndConfirm(homePage.getRemindInnerLink(feedbackSessionClosed.getCourseId(),
+                                                             feedbackSessionClosed.getFeedbackSessionName()));
```

Comment:
Why these are repeated ?. Also now we are using 8 space indentation. You should break at higher level. 
After this `homePage.clickAndConfirm(homePage.getRemindInnerLink(`